sync/fromDisk/questions.js

```diff
@@ -45,6 +46,7 @@ function getParamsForQuestion(q) {
         workspace_sync_ignore: (q.workspaceOptions && q.workspaceOptions.syncIgnore),
         workspace_url_rewrite: (q.workspaceOptions && q.workspaceOptions.rewriteUrl),
         workspace_enable_networking: (q.workspaceOptions && q.workspaceOptions.enableNetworking),
+        workspace_environment: (q.workspaceOptions && q.workspaceOptions.environment) || {},
```

Comment:
Nit: we're running Node 14, so we can use optional chaining and nullish coalescing to make this a bit more concise:
```suggestion
        workspace_environment: q.workspaceOptions?.environment ?? {},
```